eqcorrscan/utils/despike.py

```diff
@@ -58,8 +58,8 @@ def median_filter(tr, multiplier=10, windowlength=0.5,
     # Note - might be worth finding spikes in filtered data
     filt = tr.copy()
     filt.detrend('linear')
-    filt.filter('bandpass', freqmin=10.0,
-                freqmax=(tr.stats.sampling_rate / 2) - 1)
+    # filt.filter('bandpass', freqmin=9.999,
```

Comment:
It would be good to have a separate conversation and pull request to improve despiking, so can we leave this out for now?